src/jobs/release-package.yml

```diff
@@ -194,7 +194,7 @@ steps:
   # clean up the CTC cases
   - when:
       condition:
-        equal: ['latest', <<parameters.tag>>]
+        equal: ["latest", <<parameters.tag>>]
```

Comment:
@mshanemc why the switch to double quotes?

Reply:
vscode being a jerk